### ISSUE TITLE
only works with newer limnoria

### ISSUE THREAD

**Issue by yardenac** [ncoevoet/ChanTracker#26]
You started using opSettable in 02f23df44441cd7a1d1015c225bab955397453e7 which was only introduced in https://github.com/ProgVal/Limnoria/commit/9c1d46d151ebd5feda5eafae60105a7543c282b6 so earlier versions of Limnoria will not work anymore.

Limnoria's [instructions](http://doc.supybot.aperio.fr/en/latest/use/install.html) for Ubuntu tell us to just `sudo apt-get install limnoria`, which gives us [limnoria 2018.01.25](https://packages.ubuntu.com/bionic/limnoria) on 18.04. This is too old for us. Might want to add instructions to run from pip or a PPA.

**Comment by teward** [comment]
Just going to share some bits here, but these're relevant items for you:

Relevant Links:
* https://askubuntu.com/questions/151283/why-dont-the-ubuntu-repositories-have-the-latest-versions-of-software

Relevant blurb from the Limnoria documentation GitHub from Progval (https://github.com/Limnoria/Limnoria-doc/issues/85#issuecomment-429846098):

> People on Debian and Ubuntu LTS know (or should know) that their packages are not updated to provide more stability.
> Debian has backports for users who want an updated version of Limnoria.

Note that I also asked in that bit as well to add warnings about repo versions being out of date, to the Limnoria docs.

TL;DR, Ubuntu does not update the packages regularly.  Backports are only done by community request and when the Backports team has time to dedicate to evaluating and testing backports, and all are community requested/supported.  If you need updated Limnoria to make something work, either install from PyPI or from source.

(Third party modules are not dependent on individual operating systems - just saying).

**Comment by ProgVal** [comment]
> Relevant blurb from the Limnoria documentation GitHub from Progval (Limnoria/Limnoria-doc#85 (comment)):
>> People on Debian and Ubuntu LTS know (or should know) that their packages are not updated to provide more stability.

Well, that doesn't apply here, because 18.04 is the latest version of Ubuntu.

I support PR #27 to mention this version requirement in the README.

**Comment by ncoevoet** [comment]
README has been updated, thanks teward for the pull request

**Comment by yardenac** [comment]
Thank you!!
